build/int.cloudbuild.yaml

```diff
@@ -38,6 +44,24 @@ steps:
   dir: infra
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run TestSimpleExample --stage teardown --verbose']
+
```

Comment:
future: This change seems to be doubling the test time. However, we could potentially run the test cases in parallel if we're feeling really good about the suffix namespacing.

Reply:
I'm giving this a go, but I'm not sure how cft will go with isolation of local tf caches, but we'll see.

Reply:
It works!

Reply:
For context: everything deploys in the same project, and the two cft runs don't conflict with each other. The tests needed to be updated to ensure it doesn't presume a single Cloud Run service (for instance)